release/models/optical-transport/openconfig-channel-monitor.yang

```diff
@@ -26,7 +26,13 @@ module openconfig-channel-monitor {
     channel monitor (OCM) for optical transport line system
     elements such as wavelength routers (ROADMs) and amplifiers.";
 
-  oc-ext:openconfig-version "0.4.0";
+  oc-ext:openconfig-version "0.4.1";
```

Comment:
Since this is adding a leaf and is backwards compatible, the minor revision should be updated (per https://semver.org/ versioning standards followed by OC)

```suggestion
  oc-ext:openconfig-version "0.5.0";
```

Reply:
Done, thanks for the pointer to the versioning standard.